specs/protocol/README.md

```diff
@@ -407,26 +407,14 @@ a `Close` can be sent by Alice _or_ the PFI as a reply to an RFQ or a Quote. It
-| `paymentInstructions` | [`PaymentInstructions`](#paymentinstructions) | N        | Object that describes how to pay the PFI, and how to get paid by the PFI (e.g. BTC address, payment link) |
-
 
 #### `QuoteDetails`
-| field            | data type | required | description                                                      |
-| ---------------- | --------- | -------- | ---------------------------------------------------------------- |
-| `currencyCode`   | string    | Y        | ISO 3166 currency code string                                    |
-| `amountSubunits` |           | Y        | The amount of currency expressed in the smallest respective unit |
-| `feeSubunits`    | string    | N        | The amount paid in fees                                          |
-
-
-> [!NOTE]
-> 
-> Include a section that explains `feeSubunits`. Does `amountSubUnits` _include_ `feeSubunits` or does `amountSubunits + feeSubunits = totalSubunits`?
-
-#### `PaymentInstructions`
-| field    | data type                                   | required | description                                               |
-| -------- | ------------------------------------------- | -------- | --------------------------------------------------------- |
-| `payin`  | [`PaymentInstruction`](#paymentinstruction) | N        | Link or Instruction describing how to pay the PFI.        |
-| `payout` | [`PaymentInstruction`](#paymentinstruction) | N        | Link or Instruction describing how to get paid by the PFI |
+| field                | data type                                  | required | description                                                                                                                                                                                        |
+|----------------------|--------------------------------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `currencyCode`       | string                                     | Y        | ISO 3166 currency code string                                                                                                                                                                      |
+| `amountSubunits`     | string                                     | Y        | The amount of currency paid out from or paid in to Alice in the smallest respective unit                                                                                                           |
+| `feeSubunits`        | string                                     | N        | The amount paid in fees expressed in the smallest respective unit. This field is strictly informative, i.e. it does not affect the total payin/payout amount stated in the `amountSubunits` field. |
```

Comment:
> it does not affect the total payin/payout amount stated in the `amountSubunits` field.

Reads as slightly confusing because it _does_ affect the `amountSubunits` but _does not_ affect the "total". I think it's better to include the nuance here in the `amountSubunits` row, [like you had previously](https://github.com/TBD54566975/tbdex/commit/8bd8794e0c02b2e4b4ba7ae100eb7f1fb3bfeaad), and then also include `This field is strictly informative` in the `feeSubunits. So, like this...

```suggestion
| `amountSubunits`     | string                                     | Y        | The amount of payment, `feeSubunits` included, expressed in the smallest respective unit.                                                                                                           |
| `feeSubunits`        | string                                     | N        | The amount of fee payment expressed in the smallest respective unit; this field is strictly informative as `amountSubunits` includes `feeSubunits`. |
```

(Small side comment, we should try to be consistent in including or excluding periods `"."` at the end of each description)

Reply:
There is a proposed distinction that came up at office hours where `amountSubunits` does not always include `feeSubunits`. `amountSubunits` includes `feeSubunits` on payin. On payout, `amountSubunits` does NOT include `feeSubunits`. In both cases,  `amountSubunits` represents exactly what Alice will receive/pay. Alice can be unaware of `feeSubunits`.

My current opinion is that `amountSubunits` should never include `feeSubunits`, but I'll stick with the above if there is consensus. @mistermoe @jiyoontbd @corcillo @phoebe-lew Am I representing the decision from office hours accurately? Is it still your preference?

Reply:
> amountSubunits includes feeSubunits on payin. On payout, amountSubunits does NOT include feeSubunits. In both cases, amountSubunits represents exactly what Alice will receive/pay. Alice can be unaware of feeSubunits.

yes, i think this is accurate with respect to what was discussed during office hour.

> My current opinion is that amountSubunits should never include feeSubunits, but I'll stick with the above if there is consensus.

hmm, maybe we are focusing too much at the protocol layer on what is _presented visually_ to alice to make sure she understands exactly how much the PFI is asking, and how much she should receive.

it may be easier to remember / reason about in PFI implementation if we aligned on **never** or **always** including `feeSubunits` in `amountSubunits` for both payin and payout.

Reply:
> it may be easier to remember / reason about in PFI implementation if we aligned on never or always including feeSubunits in amountSubunits for both payin and payout.

I second this, as not being consistent expresses a preference at the protocol level (b/c the inverse could also be true, which would prioritize comprehension from the PFI's-side)

Reply:
> if we aligned on never or always

IMO the `amount` should **never** include the fee. This is how I've always seen it structured in fintech in the past.